plugins/workspaces/commands/delete.md

```
@@ -0,0 +1,114 @@
+---
+description: Delete a workspace and its git worktrees
+argument-hint: <workspace-name>
+---
+
+## Name
+workspaces:delete
+
+## Synopsis
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)